lib/inputs/index.js

```diff
@@ -80,12 +80,20 @@ module.exports.BigqueryInputs = class BigqueryInputs extends Inputs {
 }
 module.exports.PingbackInputs = class PingbackInputs extends Inputs {
   constructor(records) {
-    super(
-      records,
-      new Pingbacks(records),
-      new LegacyAdzerkImpressions(records),
-      new LegacyAdzerkPingbacks(records)
-    );
+    if (process.env.NO_3RD_PARTY_PINGBACKS) {
```

Comment:
So this is so we can use an ENV VAR to turn off 3rd party pingbacks w/o a code change - cool